connect/service_test.go

```diff
@@ -288,3 +290,15 @@ func TestService_HasDefaultHTTPResolverFromAddr(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, expected, got)
 }
+
+func getSubjects(cp *x509.CertPool) string {
```

Comment:
Maybe drop a TODO here for the future to reevaluate the kind of comparison we _should_ do here.